include/tscore/ink_time.h

```diff
@@ -43,7 +45,17 @@
 
  *===========================================================================*/
 
-typedef time_t ink_time_t;
+using ink_time_t  = time_t;
+using ts_clock    = std::chrono::system_clock;
+using ts_time     = ts_clock::time_point;
+using ts_hr_clock = std::chrono::high_resolution_clock;
+using ts_hr_time  = ts_hr_clock::time_point;
+
+using ts_seconds      = std::chrono::seconds;
+using ts_milliseconds = std::chrono::milliseconds;
```

Comment:
Any reason not to add ts_microseconds while we're at it? That's not an uncommon unit. In fact there's a reference to microseconds later in this file.

Reply:
oh, haha. Looks like you're other PR removes microseconds. I suppose if it's unused we might as well not add it.

Reply:
It's trivial to add if it comes up. So far the work I've done using this, it hasn't.